programs/zstdcli.c

```diff
@@ -693,6 +699,7 @@ int main(int const argCount, const char* argv[])
                 if (!strcmp(argument, "--rm")) { FIO_setRemoveSrcFile(prefs, 1); continue; }
                 if (!strcmp(argument, "--priority=rt")) { setRealTimePrio = 1; continue; }
                 if (!strcmp(argument, "--output-dir-flat")) {nextArgumentIsOutDirName=1; lastCommand=1; continue; }
+                if (!strcmp(argument, "--show-default-cparams")) { showDefaultCParams = 1; continue; }
```

Comment:
Consider documenting this new command within `man zstd`, hence in `zstd.1.md`.

Very advanced command like this one don't need to be listed by `zstd -h`.

Reply:
Okay, I'll add it to the man